.changeset/honest-walls-type.md

```diff
@@ -0,0 +1,10 @@
+---
+'@vanilla-extract/babel-plugin': minor
+'@vanilla-extract/css': minor
+'@vanilla-extract/esbuild-plugin': minor
+'@vanilla-extract/webpack-plugin': minor
+---
+
+Add `packageName` to filescopes
```

Comment:
Does this need to be reframed for the consumer perspective? e.g. `Ensure generated hashes are scoped by package`